saleor/app/tasks.py

```diff
@@ -22,25 +22,25 @@ def install_app_task(job_id, activate=False):
         msg = ", ".join([f"{name}: {err}" for name, err in e.message_dict.items()])
         app_installation.message = msg
     except AppInstallationError as e:
-        logger.warning("Failed to install app. Error: %s", e)
+        logger.warning(f"Failed to install app. Error: {e}")
```

Comment:
F-string is not suitable for logging, please revert to lazy formatting.

Reply:
Ah! Sorry, I didn't know that. Looks like it's still in debate and has an open issue https://github.com/python/cpython/issues/90358 for it. 
Thank you, @8r2y5, for letting me know. I've reverted the f-strings in log messages here: 67e6954